js/validation/validationSignIn.js

```diff
@@ -0,0 +1,62 @@
+const validationSignIn = () => {
+  const email = document.getElementById('email')
+  const password = document.getElementById('password')
+  const errorMessageEmail = document.querySelector('.error-message__email')
+  const errorMessagePassword = document.querySelector(
+    '.error-message__password'
+  )
+
+  if (email.value === '') {
+    email.classList.add('form__input_error')
+    errorMessageEmail.innerHTML = 'Please enter your email.'
+    errorMessageEmail.classList.add('form__text_error')
+    return false
+  } else {
+    email.classList.remove('form__input_error')
+    errorMessageEmail.innerHTML = ''
+  }
+
+  if (!emailIsValid(email.value)) {
+    email.classList.add('form__input_error')
+    errorMessageEmail.innerHTML = 'Please enter a valid email.'
+    errorMessageEmail.classList.add('form__text_error')
+
+    return false
+  } else {
+    email.classList.remove('form__input_error')
+    errorMessageEmail.innerHTML = ''
+  }
```

Comment:
Remove duplicates in code, this functional can be moved into funtion with params email and errorMessageEmail / and for password the same function with password and errorMessagePassword, cause both do the same.